### PR TITLE
TypeScript source maps debugging + code coverage report correct

### PR DESCRIPTION
**Pull request by clbond**
- Fix source maps when debugging TypeScript code
- Fix code coverage report so that it generates correct numbers and uses TypeScript source maps correctly

Connected to rangle/rangle-starter#92

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -70,3 +89,20 @@ module.exports = (config) => {
     singleRun: true,
   });
 };
+
+function combinedLoaders() {
+  return Object.keys(loaders).reduce(function reduce(aggregate, k) {
+    switch (k) {
+    case 'tslint': // intolerably slow
```
Comment (SethDavenport) [inline karma.conf.js]: Same comment as on the other one

**Comment by SethDavenport** [comment]
```
START:
15 06 2016 22:24:40.105:INFO [karma]: Karma v0.13.22 server started at http://localhost:9999/
15 06 2016 22:24:40.112:INFO [launcher]: Starting browser Chrome
15 06 2016 22:24:41.740:INFO [Chrome 43.0.2357 (Linux 0.0.0)]: Connected on socket /#bcQ-WYW_9Nj4apWFAAAA with id 20827212
Chrome 43.0.2357 (Linux 0.0.0) ERROR
  Uncaught TypeError: Object.assign is not a function
  at /home/ubuntu/angular-redux-starter/src/tests.entry.ts:50

Finished in 0.234 secs / 0 secs

SUMMARY:
✔ 0 tests completed
npm ERR! Test failed.  See above for more details.
```

**Comment by SethDavenport** [comment]
This is likely due to the ancient version of Chrome on CircleCI

However it's kind of a good think because it catches missing polyfills in the test flow.

**Comment by clbond** [comment]
I should not be setting __DEV__ and __PRODUCTION__ myself anyway; that should come from Webpack. Fixed now

**Comment by SethDavenport** [comment]
:shipit:

**Pull request merged**